src/main/java/com/example/DsStore/entities/Customer.java

```diff
@@ -0,0 +1,21 @@
+package com.example.DsStore.entities;
+
+
+import jakarta.persistence.*;
```

Comment:
IDEs have a default behaviour of replacing multiple imports of same library by star (*), also called as wildcards.
Could you search online and see what are people's opinion on this?

Let me know here.

Reply:
There are both advantages and disadvantages of wildcards:
adv:- clean code
dis adv:- name conflicts, *  import all the classes under that package most of which may not be even.
Hence most people **view wildcards as bad coding practice.**